src/vs/platform/product/common/product.ts

```diff
@@ -55,21 +55,6 @@ else {
 	// Built time configuration (do NOT modify)
 	product = { /*BUILD->INSERT_PRODUCT_CONFIGURATION*/ } as IProductConfiguration;
 
-	// Running out of sources
```

Comment:
I am not sure we can just remove this block, if any code depends on `product.ts` directly (and not product service), the behaviour would be different now?